design/sketch.webidl

```diff
@@ -520,6 +529,8 @@ dictionary WebGPURenderPassDepthStencilAttachmentDescriptor {
 dictionary WebGPURenderPassDescriptor {
     sequence<WebGPURenderPassColorAttachmentDescriptor> colorAttachments;
     WebGPURenderPassDepthStencilAttachmentDescriptor depthStencilAttachment;
+    // Number of MSAA samples for rasterization
+    u32 samples;
```

Comment:
Isn't that already encoded in the textures themselves? We have to check that all textures have the same sample count.

Reply:
that's the point: we'll go through the actual textures and check their sample counts:
  - if it matches, good
  - it this one is higher than 1, but the texture sample count is 1, then this is a resolve attachment
  - otherwise, it's an error

Reply:
What if `samples` is 2 and the texture has a sample count of 4? I'm not sure that having magic values is the best way to describe multisample resolves. Didn't we have a whole GitHub investigation on multisampling?

Reply:
I am also confused by the samples field.  Having one `samples` field prevents us from resolving some of the attachments to 4 and others to 2.  I think we also need additional data to know which textures are resolve sources and which ones are resolve destinations.